docs/list-of-diagnostics.md

```diff
@@ -41,6 +41,16 @@ if desired.
 | `EXTEXP0016` | Hosting integration testing experiments |
 | `EXTEXP0017` | Contextual options experiments |
 
+# Obsoletions
+
+At some point in its lifecycle APIs become obsolete. If you use obsolete APIs, you will get one of the diagnostic shown below.
+
+You may continue using obsolete APIs in your application, but we advise exploring proposed alternatives which you will find in the obsoletion message.
+
+| Diagnostic ID     | Description |
+| :---------------- | :---------- |
+| `EXTOBS0001` | IResourceUtilizationPublisher obsoletions. Please use observable instruments from [here](https://github.com/dotnet/extensions/blob/6e0195db97d66cdf082e32799858c34bf5fd0cdb/src/Libraries/Microsoft.Extensions.Diagnostics.ResourceMonitoring/ResourceUtilizationInstruments.cs). |
```

Comment:
The message should be the same as in the [DiagnosticIds.cs](https://github.com/dotnet/extensions/pull/5360/files#diff-771ed0a6234e6f7e731f2b52210e0cbb7e9484f2179274e4bc7ff38dfea224e8). Though here, we can add code snippets, links to other docs or samples which help a developer to migrate off the obsolete code.

Reply:
Updated